ios/Classes/Types/UserScript.swift

```diff
@@ -10,8 +10,19 @@ import WebKit
 
 public class UserScript : WKUserScript {
     var groupName: String?
+
+    var contentWorldWrapper: Any?
     @available(iOS 14.0, *)
-    lazy var contentWorld: WKContentWorld = WKContentWorld.page
+    var contentWorld: WKContentWorld {
+      get {
+        if let value = contentWorldWrapper as? WKContentWorld {
```

Comment:
To keep lazy behaviour i'd suggest the following implementation
```
if contentWorldWrapper == nil {
    contentWorldWrapper = WKContentWorld.page
}
return contentWorldWrapper as! WKContentWorld
```

, but yours is safer